python/lib/dependabot/python/file_updater/pipfile_file_updater.rb

```diff
@@ -233,20 +233,20 @@ def post_process_lockfile(updated_lockfile_content)
 
         def generate_updated_requirements_files
           req_content = run_pipenv_command(
-            "pyenv exec pipenv lock -r"
+            "pyenv exec pipenv requirements"
```

Comment:
I think we'll instead want to long-term go with:
* https://github.com/dependabot/dependabot-core/issues/6836